docs/pages/distribution/publishing-websites-webpack.mdx

```diff
@@ -0,0 +1,337 @@
+
+## Hosting on third-party services
+
+### Netlify
+
+[Netlify](https://www.netlify.com/) is a mostly-unopinionated platform for deploying web apps. This has the highest compatibility with Expo web apps as it makes few assumptions about the framework.
+
+#### Manual deployment with the Netlify CDN
+
+<Step label="1">
+
+Install the Netlify CLI by running the following command:
+
+<Terminal cmd={['$ npm install -g netlify-cli']} />
+
+</Step>
+
+<Step label="2">
+
+Configure redirects for single-page applications.
+
+If your app implements any navigation, you'll need to configure Netlify to redirect requests to the single **web-build/index.html** file. This can be done in Netlify by creating a **./public/\_redirects** file and redirecting all requests to **/index.html**.
+
+Navigate inside the **web-build** directory and run the following command to create **\_redirects** file with follwing rule:
+
+```sh web/_redirects
+/*    /index.html   200
+```
+
+If you modify this file, you must rebuild your project with `npx expo export:web` to have it safely copied into the **web-build** directory.
+
+</Step>
+
+<Step label="3">
+
+Deploy the web build folder by running the following command:
+
+<Terminal cmd={['$ netlify deploy --dir web-build']} />
+
+You'll see a URL that you can use to view your project online.
+
+</Step>
+
+#### Continuous delivery
+
+Netlify can also build and deploy when you push to git or open a new pull request:
+
+- [Start a new Netlify project](https://app.netlify.com/signup).
+- Pick your Git hosting service and select your repository.
+- Click **Build your site**.
+
+### Vercel
+
+[Vercel](https://vercel.com/) has a single-command deployment flow.
+
+<Step label="1">
+
+Install the [Vercel CLI](https://vercel.com/docs/cli).
+
+<Terminal cmd={['$ npm install -g vercel@latest']} />
+
+</Step>
+<Step label="2">
+  Configure redirects for single-page applications.
+  Create a **vercel.json** file at the root of your app and add the following:
+
+  ```json vercel.json
+  {
+    "buildCommand": "expo export:web",
+    "outputDirectory": "web-build",
+    "devCommand": "expo",
+    "cleanUrls": true,
+    "framework": null,
+    "rewrites": [
+      {
+        "source": "/:path*",
+        "destination": "/"
+      }
+    ]
+  }
+  ```
+
+</Step>
+
+<Step label="3">
+  Deploy the website.
+
+{' '}
+
+<Terminal cmd={['$ vercel']} />
+
+You'll now see a URL that you can use to view your project online. Paste that URL into your browser when the build is complete, and you'll see your deployed app.
+
+</Step>
+
+### AWS Amplify Console
+
+The [AWS Amplify Console](https://console.amplify.aws) provides a Git-based workflow for continuously deploying and hosting full-stack serverless web apps. Amplify deploys your PWA from a repository instead of from your computer. In this guide, we'll use a GitHub repository. Before starting, [create a new repo on GitHub](https://github.com/new).
```

Comment:
```suggestion
The [AWS Amplify Console](https://console.amplify.aws) provides a Git-based workflow for continuously deploying and hosting full-stack serverless web apps. Amplify deploys your PWA from a repository instead of your computer, so you must use a GitHub repository. Before starting, [create a new repo on GitHub](https://github.com/new).
```